packages/compiler-dom/src/parserOptionsMinimal.ts

```diff
@@ -73,9 +79,7 @@ export const parserOptionsMinimal: ParserOptions = {
       if (tag === 'textarea' || tag === 'title') {
         return TextModes.RCDATA
       }
-      if (
-        /^(?:style|xmp|iframe|noembed|noframes|script|noscript)$/i.test(tag)
```

Comment:
it should be `.test(tag.toLowerCase())`

Reply:
The `makeMap` function takes an optional `boolean` argument `expectsLowerCase` which is just for this use case.